.ci/jenkins/Jenkinsfile

```diff
@@ -161,6 +161,20 @@ try{
         }*/ // EXTREMELY SLOW, INVESTIGATE
         parallel builders
     }
+    if (env.BRANCH_NAME == "develop") {
+        // Deploy snapshot to test pypi if branch develop
+        node("Linux") {
+           stage("Deploy snapshot to pypitesting"){
+              checkout scm
+              withCredentials([string(credentialsId: 'TWINE_USERNAME', variable: 'TWINE_USERNAME'),
```

Comment:
are you using the same credentials for test.pypi.org and pypi.org?

Reply:
That is about Jenkins, the user/pass are stored in a "credential" storage.